src/python/platforms/fuchsia/common.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Google LLC
```

Comment:
let's avoid naming things "common" and be more specific. probably call this file `errors` or something similar?